mapstore-dynamodb-test/src/test/scala/net/spals/appbuilder/mapstore/dynamodb/DynamoDBMapStorePluginIT.scala

```diff
@@ -178,5 +193,30 @@ class DynamoDBMapStorePluginIT {
                    expectedResults: List[Map[String, AnyRef]]) {
     assertThat(mapStorePlugin.getItems(rangeTableName, storeKey, defaultOptions()),
       contains[java.util.Map[String, AnyRef]](expectedResults.map(_.asJava): _*))
+    assertThat(dynamoDBTracer.finishedSpans(), contains[MockSpan](dynamoDBSpan(dynamoDBEndpoint, "POST")))
   }
 }
+
+private object DynamoDBSpanMatcher {
+
+  def dynamoDBSpan(url: String, method: String): DynamoDBSpanMatcher =
+    DynamoDBSpanMatcher(url, method)
+}
+
+private case class DynamoDBSpanMatcher(url: String, method: String) extends TypeSafeMatcher[MockSpan] {
+
+  override def matchesSafely(mockSpan: MockSpan): Boolean = {
+    Matchers.hasEntry[String, AnyRef]("component", "java-aws-sdk").matches(mockSpan.tags()) &&
```

Comment:
Shouldn't you use Matchers.allOf(...) which applies and's for you instead of &&?